<launch>
   <rosparam file="$(find pr2_diagnostic_controller)/diagnostic_controller.yaml" command="load" />

   <node pkg="pr2_controller_manager" type="spawner" args="diagnostic_controller" name="diagnostic_controller_spawner" />
</launch>
